code/modules/events/queen_mothers_blessing.dm

```diff
@@ -0,0 +1,36 @@
+#define QM_HEAL_AMOUNT 250
+
+/datum/round_event_control/queen_mothers_blessing
+	name = "Queen Mother's Blessing"
+	typepath = /datum/round_event/queen_mothers_blessing
+	weight = 15
+	earliest_start = 5 MINUTES
+
+	gamemode_blacklist = list("Combat Patrol","Civil War")
+
+/datum/round_event_control/queen_mothers_blessing/can_spawn_event(players_amt, gamemode)
+	if(length(GLOB.alive_xeno_list) <= 0)
+		return FALSE
+	return ..()
+
+/datum/round_event/queen_mothers_blessing/start()
+	for(var/mob/living/carbon/xenomorph/X in shuffle(GLOB.alive_xeno_list))
+		if(isminion(X))
+			continue
+		bless_xeno(X)
+		break
+
+/datum/round_event/queen_mothers_blessing/proc/bless_xeno(mob/living/carbon/xenomorph/X)
+	X.evolution_stored = X.evolution_stored + (X.evolution_stored * 1.75)
```

Comment:
```suggestion
	X.evolution_stored += X.evolution_stored * 1.75
```
Also this is 175% of their evolution stored and I'm 99% sure it doesn't go over 100%.

Reply:
Weirdly enough the evolution can go over 100% if forced, during testing I saw it hit 1000% percent after multiple forced consecutive uses of Queen Mother's blessing on a single xeno. It still resets to 0 after an evolution though, no matter how large the number.